.dotnet/src/Custom/FineTuning/Internal/InternalFineTuneChatCompletionRequestFunctionMessage.Serialization.cs

```diff
@@ -0,0 +1,43 @@
+using OpenAI.Chat;
```

Comment:
Fine-tuning is currently protocol-only. Why is any customization needed for this class?

Reply:
I'm not sure why it was done, but this spec update added model types for FT input; this is new, and since the models get picked up we need to do something with them (in this case, just move/internalize them) to stop the model factory from breaking.